languages/python/django-oso/tests/test_django_oso.py

```diff
@@ -133,28 +133,38 @@ def test_route_authorization(client, settings, simple_policy):
 def test_partial(rf, settings, partial_policy):
     from test_app.models import Post
 
-    Post(name="test", is_private=False, timestamp=1).save()
-    Post(name="test_past", is_private=False, timestamp=-1).save()
-    Post(name="test_public", is_private=False, timestamp=1).save()
-    Post(name="test_private", is_private=True, timestamp=1).save()
-    Post(name="test_private_2", is_private=True, timestamp=1).save()
+    posts = [
+        Post(name="test", is_private=False, timestamp=1).save(),
+        Post(name="test_past", is_private=False, timestamp=-1).save(),
+        Post(name="test_public", is_private=False, timestamp=1).save(),
+        Post(name="test_private", is_private=True, timestamp=1).save(),
+        Post(name="test_private_2", is_private=True, timestamp=1).save(),
+        Post(name="test_option", is_private=True, timestamp=0, option=True).save(),
```

Comment:
```suggestion
        Post(name="test_option", is_private=False, timestamp=1, option=True).save(),
```
wouldn't this assure us that this Post isn't turned up by the authorize_filter on 148 b/c of `option` not being `nil` (as opposed to `is_private` being `True` or `timestamp` not being greater than `0`)?

Reply:
Yup, thanks.